### ISSUE TITLE
Bug: Enforcement failed (input/output error) 

### ISSUE THREAD

**Issue by szx0112** [abraunegg/onedrive#2596]
### Describe the bug

I run the command: onedrive --monitor
it works great but only uploaded partial files in the target folder (300G/900G), when I type in the onedrive --monitor command again, it always return the following errors:
```
$ onedrive --monitor
Configuration file successfully loaded
Configuring Global Azure AD Endpoints
Continuing the upload session ...
Uploading   0% |o                                       |   ETA   00:00:00 std.exception.ErrnoException@/usr/include/dmd/phobos/std/stdio.d(1205): Enforcement failed (Input/output error)
----------------
/usr/include/dmd/phobos/std/exception.d:521 @safe noreturn std.exception.bailOut!(std.exception.ErrnoException).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x558e394971d0]
/usr/include/dmd/phobos/std/exception.d:442 @safe bool std.exception.enforce!(std.exception.ErrnoException).enforce!(bool).enforce(bool, lazy const(char)[], immutable(char)[], ulong) [0x558e39523c6b]
/usr/include/dmd/phobos/std/stdio.d:1205 @safe void[] std.stdio.File.rawRead!(void).rawRead(void[]) [0x558e39523b39]
src/onedrive.d:955 @safe ulong onedrive.OneDriveApi.uploadFragment(const(char)[], immutable(char)[], long, long, long).__lambda9!(void[]).__lambda9(void[]) [0x558e39598b88]
??:? ulong function(void[]) std.net.curl.Curl.onSend(ulong delegate(void[])).__lambda2 [0x558e3963f39b]
??:? extern (C) ulong std.net.curl.Curl._sendCallback(char*, ulong, ulong, void*) [0x558e3963f7b3]
??:? [0x7febf2a4aefd]
??:? [0x7febf2a4b6f2]
??:? [0x7febf2a565ba]
??:? curl_multi_perform [0x7febf2a57b13]
??:? curl_easy_perform [0x7febf2a4d993]
??:? int std.net.curl.Curl.perform(std.typecons.Flag!("throwOnError").Flag) [0x558e3963f079]
??:? int std.net.curl.HTTP.perform(std.typecons.Flag!("throwOnError").Flag) [0x558e3963cbcb]
src/onedrive.d:1486 std.json.JSONValue onedrive.OneDriveApi.perform() [0x558e3959ac54]
src/onedrive.d:958 std.json.JSONValue onedrive.OneDriveApi.uploadFragment(const(char)[], immutable(char)[], long, long, long) [0x558e39598a1f]
src/upload.d:204 std.json.JSONValue upload.UploadSession.upload() [0x558e395c33b5]
src/sync.d:482 void sync.SyncEngine.init() [0x558e395a5c1b]
src/main.d:1772 bool main.initSyncEngine(sync.SyncEngine) [0x558e3958f298]
src/main.d:1198 _Dmain [0x558e3958c8da]
std.net.curl.CurlException@std/net/curl.d(4401): A libcurl function was given a bad argument on handle null
----------------
/usr/include/dmd/phobos/std/exception.d:521 pure @safe noreturn std.exception.bailOut!(std.net.curl.CurlException).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x558e39568427]
/usr/include/dmd/phobos/std/exception.d:442 pure @safe bool std.exception.enforce!(std.net.curl.CurlException).enforce!(bool).enforce(bool, lazy const(char)[], immutable(char)[], ulong) [0x558e3956839b]
??:? void std.net.curl.Curl._check(int) [0x558e3963ec29]
??:? void std.net.curl.Curl.clear(etc.c.curl.CurlOption) [0x558e3963efde]
??:? void std.net.curl.HTTP.clearRequestHeaders() [0x558e3963cd63]
src/onedrive.d:500 void onedrive.OneDriveApi.shutdown() [0x558e39596601]
src/main.d:84 _Dmain [0x558e3958ea2c]
```

### Operating System Details

```shell
5.4.0-150-generic
ubuntu 18.04.6 LTS
```

### Client Installation Method

From Source

### OneDrive Account Type

Personal

### What is your OneDrive Application Version

v2.4.25-13-g1a88d33

### What is your OneDrive Application Configuration

```shell
onedrive --display-config
Configuration file successfully loaded
onedrive version                             = v2.4.25-13-g1a88d33
Config path                                  = /home/szx/.config/onedrive
Config file found in config path             = true
Config option 'sync_dir'                     = /media/szx/Data1/shared_onedrive_folder
Config option 'enable_logging'               = false
Config option 'log_dir'                      = /var/log/onedrive/
Config option 'disable_notifications'        = false
Config option 'min_notify_changes'           = 5
Config option 'skip_dir'                     = 
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ~*|.~*|*.tmp
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 300
Config option 'monitor_log_frequency'        = 6
Config option 'monitor_fullscan_frequency'   = 12
Config option 'read_only_auth_scope'         = false
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'cleanup_local_files'          = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'space_reservation'            = 52428800
Config option 'application_id'               = 
Config option 'azure_ad_endpoint'            = 
Config option 'azure_tenant_id'              = common
Config option 'user_agent'                   = 
Config option 'force_http_11'                = false
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 0
Config option 'operation_timeout'            = 3600
Config option 'dns_timeout'                  = 60
Config option 'connect_timeout'              = 10
Config option 'data_timeout'                 = 600
Config option 'ip_protocol_version'          = 0
Config option 'sync_root_files'              = false
Selective sync 'sync_list' configured        = false
Config option 'sync_business_shared_folders' = false
Business Shared Folders configured           = false
Config option 'webhook_enabled'              = false
```

### What is your 'curl' version

```shell
curl 7.58.0 (x86_64-pc-linux-gnu) lib curl/7.58.0 OpenSSL/1.1.1 zlib/1.2.11 libidn2/2.0.4 libels/0.19.1 (+libidn2/2.0.4) nghttp2/1.30.0 librtmp/2.3
Release-Date: 2018-01-24
```

### Where is your 'sync_dir' located

Local

### What are all your system 'mount points'

```shell
mount
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
udev on /dev type devtmpfs (rw,nosuid,relatime,size=264123320k,nr_inodes=66030830,mode=755)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,noexec,relatime,size=52829884k,mode=755)
/dev/nvme0n1 on / type ext4 (rw,relatime,errors=remount-ro)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k)
tmpfs on /sys/fs/cgroup type tmpfs (ro,nosuid,nodev,noexec,mode=755)
cgroup on /sys/fs/cgroup/unified type cgroup2 (rw,nosuid,nodev,noexec,relatime,nsdelegate)
cgroup on /sys/fs/cgroup/systemd type cgroup (rw,nosuid,nodev,noexec,relatime,xattr,name=systemd)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
cgroup on /sys/fs/cgroup/cpuset type cgroup (rw,nosuid,nodev,noexec,relatime,cpuset)
cgroup on /sys/fs/cgroup/rdma type cgroup (rw,nosuid,nodev,noexec,relatime,rdma)
cgroup on /sys/fs/cgroup/net_cls,net_prio type cgroup (rw,nosuid,nodev,noexec,relatime,net_cls,net_prio)
cgroup on /sys/fs/cgroup/perf_event type cgroup (rw,nosuid,nodev,noexec,relatime,perf_event)
cgroup on /sys/fs/cgroup/cpu,cpuacct type cgroup (rw,nosuid,nodev,noexec,relatime,cpu,cpuacct)
cgroup on /sys/fs/cgroup/devices type cgroup (rw,nosuid,nodev,noexec,relatime,devices)
cgroup on /sys/fs/cgroup/memory type cgroup (rw,nosuid,nodev,noexec,relatime,memory)
cgroup on /sys/fs/cgroup/freezer type cgroup (rw,nosuid,nodev,noexec,relatime,freezer)
cgroup on /sys/fs/cgroup/blkio type cgroup (rw,nosuid,nodev,noexec,relatime,blkio)
cgroup on /sys/fs/cgroup/hugetlb type cgroup (rw,nosuid,nodev,noexec,relatime,hugetlb)
cgroup on /sys/fs/cgroup/pids type cgroup (rw,nosuid,nodev,noexec,relatime,pids)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=26,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=25772)
mqueue on /dev/mqueue type mqueue (rw,relatime)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,pagesize=2M)
debugfs on /sys/kernel/debug type debugfs (rw,relatime)
configfs on /sys/kernel/config type configfs (rw,relatime)
fusectl on /sys/fs/fuse/connections type fusectl (rw,relatime)
/var/lib/snapd/snaps/gnome-3-38-2004_140.snap on /snap/gnome-3-38-2004/140 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gtk-common-themes_1534.snap on /snap/gtk-common-themes/1534 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/shutter_27.snap on /snap/shutter/27 type squashfs (ro,nodev,relatime,x-gdu.hide)
binfmt_misc on /proc/sys/fs/binfmt_misc type binfmt_misc (rw,relatime)
/var/lib/snapd/snaps/rpi-imager_520.snap on /snap/rpi-imager/520 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/cmake_507.snap on /snap/cmake/507 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core22_864.snap on /snap/core22/864 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/rpi-imager_584.snap on /snap/rpi-imager/584 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-38-2004_143.snap on /snap/gnome-3-38-2004/143 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/pycharm-community_360.snap on /snap/pycharm-community/360 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-characters_795.snap on /snap/gnome-characters/795 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/cmake_580.snap on /snap/cmake/580 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/libspot_11.snap on /snap/libspot/11 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-28-1804_194.snap on /snap/gnome-3-28-1804/194 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core18_2812.snap on /snap/core18/2812 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-characters_791.snap on /snap/gnome-characters/791 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core20_2015.snap on /snap/core20/2015 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-34-1804_90.snap on /snap/gnome-3-34-1804/90 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/zoom-client_218.snap on /snap/zoom-client/218 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-42-2204_141.snap on /snap/gnome-42-2204/141 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-calculator_955.snap on /snap/gnome-calculator/955 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-system-monitor_186.snap on /snap/gnome-system-monitor/186 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-calculator_953.snap on /snap/gnome-calculator/953 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core_16091.snap on /snap/core/16091 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core18_2796.snap on /snap/core18/2796 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-logs_119.snap on /snap/gnome-logs/119 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/zoom-client_217.snap on /snap/zoom-client/217 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-logs_121.snap on /snap/gnome-logs/121 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/pycharm-community_358.snap on /snap/pycharm-community/358 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core_16202.snap on /snap/core/16202 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-28-1804_198.snap on /snap/gnome-3-28-1804/198 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/cloudcompare_208.snap on /snap/cloudcompare/208 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gtk2-common-themes_13.snap on /snap/gtk2-common-themes/13 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-3-34-1804_93.snap on /snap/gnome-3-34-1804/93 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-system-monitor_184.snap on /snap/gnome-system-monitor/184 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gtk-common-themes_1535.snap on /snap/gtk-common-themes/1535 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/repeat_40.snap on /snap/repeat/40 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core22_1033.snap on /snap/core22/1033 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/bare_5.snap on /snap/bare/5 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/gnome-42-2204_132.snap on /snap/gnome-42-2204/132 type squashfs (ro,nodev,relatime,x-gdu.hide)
/var/lib/snapd/snaps/core20_2105.snap on /snap/core20/2105 type squashfs (ro,nodev,relatime,x-gdu.hide)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,size=52829880k,mode=700,uid=1000,gid=1000)
gvfsd-fuse on /run/user/1000/gvfs type fuse.gvfsd-fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
/dev/sdc1 on /media/szx/Data1 type fuseblk (rw,nosuid,nodev,relatime,user_id=0,group_id=0,default_permissions,allow_other,blksize=4096,uhelper=udisks2)
/dev/fuse on /run/user/1000/doc type fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
```

### What are all your local file system partition types

```shell
lsblk -f
NAME    FSTYPE  LABEL       UUID                                 MOUNTPOINT
loop0   squashf                                                  /snap/gnome-3-3
loop1   squashf                                                  /snap/gtk-commo
loop2   squashf                                                  /snap/shutter/2
loop3   squashf                                                  /snap/rpi-image
loop4   squashf                                                  /snap/cmake/507
loop5   squashf                                                  /snap/core22/86
loop6   squashf                                                  /snap/rpi-image
loop7   squashf                                                  /snap/pycharm-c
loop8   squashf                                                  /snap/gnome-3-3
loop9   squashf                                                  /snap/gnome-cha
loop10  squashf                                                  /snap/cmake/580
loop11  squashf                                                  /snap/libspot/1
loop12  squashf                                                  /snap/gnome-3-2
loop13  squashf                                                  /snap/core18/28
loop14  squashf                                                  /snap/core20/20
loop15  squashf                                                  /snap/gnome-cha
loop16  squashf                                                  /snap/zoom-clie
loop17  squashf                                                  /snap/gnome-3-3
loop18  squashf                                                  /snap/gnome-42-
loop19  squashf                                                  /snap/gnome-cal
loop20  squashf                                                  /snap/gnome-cal
loop21  squashf                                                  /snap/gnome-sys
loop22  squashf                                                  /snap/core/1609
loop23  squashf                                                  /snap/core18/27
loop24  squashf                                                  /snap/gnome-log
loop25  squashf                                                  /snap/zoom-clie
loop26  squashf                                                  /snap/gnome-log
loop27  squashf                                                  /snap/pycharm-c
loop28  squashf                                                  /snap/core/1620
loop29  squashf                                                  /snap/gnome-3-2
loop30  squashf                                                  /snap/cloudcomp
loop31  squashf                                                  /snap/gtk2-comm
loop32  squashf                                                  /snap/gnome-3-3
loop33  squashf                                                  /snap/gnome-sys
loop34  squashf                                                  /snap/gtk-commo
loop35  squashf                                                  /snap/repeat/40
loop36  squashf                                                  /snap/core22/10
loop37  squashf                                                  /snap/bare/5
loop38  squashf                                                  /snap/gnome-42-
loop39  squashf                                                  /snap/core20/21
sda                                                              
├─sda1  vfat    DellUtility 5450-4444                            
├─sda2  ntfs    RECOVERY    7C0ECE9B0ECE4DB8                     
└─sda3  ntfs    OS          300AD2FF0AD2C150                     
sdb                                                              
└─sdb1  ntfs    Data2       30D587292384BC8A                     
sdc                                                              
└─sdc1  ntfs    Data        1618019D18017D4D                     /media/szx/Data
sr0                                                              
nvme0n1 ext4                4e9c83a9-84e8-4568-8a63-67d1f479f3b5 /
```

### How do you use 'onedrive'

try to upload all files in a folder in my ubuntu to onedrive for sync

### Steps to reproduce the behaviour

onedrive --monitor
onedrive --synchronize

### Complete Verbose Log Output

```shell
szx@szx19:~$ onedrive --synchronize --verbose
Using 'user' Config Dir: /home/szx/.config/onedrive
Using 'system' Config Dir: /etc/onedrive
Configuration file successfully loaded
Checking Application Version ...
Initializing the OneDrive API ...
Configuring Global Azure AD Endpoints
Using Curl defaults for all HTTP operations
Opening the item database ...
All operations will be performed in: /media/szx/Data1/inline_pipe_project
Application version: v2.4.25-13-g1a88d33
Account Type: personal
Default Drive ID: 8555045b2e2ef860
Default Root ID: 8555045B2E2EF860!112
Remaining Free Space: 782328292966
Fetching details for OneDrive Root
OneDrive Root exists in the database
Trying to restore the upload session ...
Continuing the upload session ...
Uploading   0% |o                                       |   ETA   00:00:00 std.exception.ErrnoException@/usr/include/dmd/phobos/std/stdio.d(1205): Enforcement failed (Input/output error)
----------------
/usr/include/dmd/phobos/std/exception.d:521 @safe noreturn std.exception.bailOut!(std.exception.ErrnoException).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x560b2e20a1d0]
/usr/include/dmd/phobos/std/exception.d:442 @safe bool std.exception.enforce!(std.exception.ErrnoException).enforce!(bool).enforce(bool, lazy const(char)[], immutable(char)[], ulong) [0x560b2e296c6b]
/usr/include/dmd/phobos/std/stdio.d:1205 @safe void[] std.stdio.File.rawRead!(void).rawRead(void[]) [0x560b2e296b39]
src/onedrive.d:955 @safe ulong onedrive.OneDriveApi.uploadFragment(const(char)[], immutable(char)[], long, long, long).__lambda9!(void[]).__lambda9(void[]) [0x560b2e30bb88]
??:? ulong function(void[]) std.net.curl.Curl.onSend(ulong delegate(void[])).__lambda2 [0x560b2e3b239b]
??:? extern (C) ulong std.net.curl.Curl._sendCallback(char*, ulong, ulong, void*) [0x560b2e3b27b3]
??:? [0x7f97fece7efd]
??:? [0x7f97fece86f2]
??:? [0x7f97fecf35ba]
??:? curl_multi_perform [0x7f97fecf4b13]
??:? curl_easy_perform [0x7f97fecea993]
??:? int std.net.curl.Curl.perform(std.typecons.Flag!("throwOnError").Flag) [0x560b2e3b2079]
??:? int std.net.curl.HTTP.perform(std.typecons.Flag!("throwOnError").Flag) [0x560b2e3afbcb]
src/onedrive.d:1486 std.json.JSONValue onedrive.OneDriveApi.perform() [0x560b2e30dc54]
src/onedrive.d:958 std.json.JSONValue onedrive.OneDriveApi.uploadFragment(const(char)[], immutable(char)[], long, long, long) [0x560b2e30ba1f]
src/upload.d:204 std.json.JSONValue upload.UploadSession.upload() [0x560b2e3363b5]
src/sync.d:482 void sync.SyncEngine.init() [0x560b2e318c1b]
src/main.d:1772 bool main.initSyncEngine(sync.SyncEngine) [0x560b2e302298]
src/main.d:1198 _Dmain [0x560b2e2ff8da]
std.net.curl.CurlException@std/net/curl.d(4401): A libcurl function was given a bad argument on handle null
----------------
/usr/include/dmd/phobos/std/exception.d:521 pure @safe noreturn std.exception.bailOut!(std.net.curl.CurlException).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x560b2e2db427]
/usr/include/dmd/phobos/std/exception.d:442 pure @safe bool std.exception.enforce!(std.net.curl.CurlException).enforce!(bool).enforce(bool, lazy const(char)[], immutable(char)[], ulong) [0x560b2e2db39b]
??:? void std.net.curl.Curl._check(int) [0x560b2e3b1c29]
??:? void std.net.curl.Curl.clear(etc.c.curl.CurlOption) [0x560b2e3b1fde]
??:? void std.net.curl.HTTP.clearRequestHeaders() [0x560b2e3afd63]
src/onedrive.d:500 void onedrive.OneDriveApi.shutdown() [0x560b2e309601]
src/main.d:84 _Dmain [0x560b2e301a2c]
```

**Comment by abraunegg** [comment]
@szx0112 
Ubuntu 18.x is no longer a supported Linux distribution. Please upgrade your platform.

Regarding your bug crash, unfortunately this will not be fixed for v2.4.x. Please test 'v2.5.0 alpha-5' once you have upgraded your Ubuntu platform.

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
